createdisk-library.sh

```diff
@@ -294,11 +294,13 @@ function download_podman() {
       mkdir -p podman-remote/mac
       curl -L https://github.com/containers/podman/releases/download/v${version}/podman-remote-release-darwin.zip -o podman-remote/mac/podman.zip
       ${UNZIP} -o -d podman-remote/mac/ podman-remote/mac/podman.zip
+      mv podman-remote/mac/podman-${version}/podman  podman-remote/mac
```

Comment:
This means `master` and the `4.8` branches are out of sync, and this  commit is not the same as the one which went into 4.8 :-/ (slightly different commit message).

Reply:
@cfergeau yes I can drop this and cherry pick what went in for 4.8

Reply:
done